integration-tests/environment-helpers/use-db.js

```diff
@@ -32,6 +32,8 @@ const keepTables = [
   "payment_provider",
   "country",
   "currency",
+  "migrations",
```

Comment:
not sure if we want to keep that. Some migrations have FF, in theory they should be cleaned up and rerun every time

Reply:
This is part of the `db.teardown()` invocation that is run between tests in a single test suite. We don't change the schema of the database between tests, rather we just clear out whatever content is in the tables between runs. 

Generally we are creating new databases (and have a db.shutdown invocation) when we have migrations that are affected by featureflags and change the database schema. 

I introduced this change due to the use of `db.entityMetadatas` since `db_` was an instance of `Connection` from Typeorm and since Typeorm didn't know anything about module migrations that differ from the core-schema of Medusa those tables were not cleaned between database runs causing spill-overs in some tables, i.e. Price-Sets leaking between tests.

Reply:
oh cool, that makes sense then. Thanks for the explanation @pKorsholm